examples/samples/deferral.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  OnMessage,
+  OnError,
+  MessagingError,
+  delay,
+  ServiceBusMessage,
+  ReceiveMode,
+  generateUuid,
+  Namespace,
+  SendableMessageInfo
+} from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const path = process.env.QUEUE_NAME || "";
+console.log("str: ", str);
+console.log("path: ", path);
+
+async function main(): Promise<void> {
+  await sendMessage();
+  await receiveMessage();
+}
+
+async function sendMessage(): Promise<void> {
+  const nsSend = Namespace.createFromConnectionString(str);
+  const sendClient = nsSend.createQueueClient(path);
+  var data = [
+    { step: 1, title: "Shop" },
+    { step: 2, title: "Unpack" },
+    { step: 3, title: "Prepare" },
+    { step: 4, title: "Cook" },
+    { step: 5, title: "Eat" }
+  ];
+  try {
+    var promises = new Set<Promise<void>>();
+    for (let index = 0; index < data.length; index++) {
+      const message: SendableMessageInfo = {
+        body: JSON.stringify(data[index]),
```

Comment:
Ok!

Reply:
I am not saying that we shouldnt send json. `data[index]` is already json. `JSON.stringify` returns a string. What I am saying is that we should send json and not string. 

I was wondering if we can send json object as is, without stringifying it. Since we set the contentType as json, I believe the SDK should return the data as json when we receive the message as well. I may be wrong in which case it is not a user friendly API.

Reply:
If I send `data[index]`, it does not return json when we receive the message, even I already set contentType as json that's why I'm sending `JSON.stringify(data[index])`, In that way It is working fine.